iscsiuio/src/unix/main.c

```diff
@@ -228,18 +232,19 @@ int get_score_path()
 int write_oom_score() 
 {
     int fd;
-	int score_path;
+	int score_path,len;
     char score[12];
 
     score_path = get_score_path();
     if (score_path < 0) {
 		snprintf(score, 12, "%d", score_path);
+		len = strlen(score);
```

Comment:
snprintf already returns the number of characters it printed.

Your code will work, but there's no reason to call another function.